Cargo.toml

```diff
@@ -2,5 +2,6 @@
 members = ["guac_core", "guac_actix", "guac_http"]
 
 [patch.crates-io]
-ethereum-types = {git="https://github.com/paritytech/primitives.git"}
-actix = { git = "https://github.com/kingoflolz/actix.git", branch = "althea-mesh" }
\ No newline at end of file
+ethereum-types = {git="https://github.com/paritytech/primitives.git", rev="6f5f681"}
```

Comment:
why are we still using the git version at all?

Reply:
I gave it a try again and it works with pin, at least from guac side.